t/run/runenv_hashseed.t

```diff
@@ -36,51 +58,75 @@ use warnings;
 
 # enable DEBUG_RUNENV if you want to see what is being returned
 # by the executed perl.
-sub my_runperl_and_capture {
-    my ($opts_hash, $cmd_array) = @_;
+sub my_runperl {
+    my ($cmd_array, $perturb, $set_seed) = @_;
+    my $opts_hash= {
+        PERL_HASH_SEED_DEBUG => 1,
+        PERL_PERTURB_KEYS => $perturb
```

Comment:
(My comment appears to have gone awol from last review, trying again)

I thought the intent was for the two runs to have identical ENV, but you're setting this as a number on one run as a string on the other. I don't understand why that isn't causing the test to fail.

Reply:
The numbers and strings are both allowed values of `PERL_PERTURB_KEYS`. The test is verifying that the name and its corresponding string behave the same.

Reply:
> The numbers and strings are both allowed values of `PERL_PERTURB_KEYS`. The test is verifying that the name and its corresponding string behave the same.

Yes, but the message from the rest of this test file is that they _should not_ behave the same, because any change in the incoming `%ENV` should cause the hash randomization to give different results. So providing an ENV key as a number one time and as a keyword the next should not give comparable results.